numba/tests/doc_examples/test_examples.py

```diff
@@ -197,6 +198,139 @@ def func_mt(*args):
             timefunc(correct, "numba (%d threads)" % nthreads, func_nb_mt, a, b)
             # magictoken.ex_no_gil.end
 
+    def test_vectorize_one_signature(self):
+        with captured_stdout():
+            # magictoken.ex_vectorize_one_signature.begin
+            from numba import vectorize, float64
+
+            @vectorize([float64(float64, float64)])
+            def f(x, y):
+                return x + y
+            # magictoken.ex_vectorize_one_signature.end
+
+    def test_vectorize_multiple_signatures(self):
+        with captured_stdout():
+            # magictoken.ex_vectorize_multiple_signatures.begin
+            from numba import vectorize, int32, int64, float32, float64
+            import numpy as np
+
+            @vectorize([int32(int32, int32),
+                        int64(int64, int64),
+                        float32(float32, float32),
+                        float64(float64, float64)])
+            def f(x, y):
+                return x + y
+            # magictoken.ex_vectorize_multiple_signatures.end
+
+            # magictoken.ex_vectorize_return_call_one.begin
+            a = np.arange(6)
+            result = f(a, a)
+            # result == array([ 0,  2,  4,  6,  8, 10])
+            # magictoken.ex_vectorize_return_call_one.end
+
+            self.assertIsInstance(result, np.ndarray)
+            correct = np.array([0, 2, 4, 6, 8, 10])
+            np.testing.assert_array_equal(result, correct)
+
+            # magictoken.ex_vectorize_return_call_two.begin
+            a = np.linspace(0, 1, 6)
+            result = f(a, a)
+            # Now, result == array([0. , 0.4, 0.8, 1.2, 1.6, 2. ])
+            # magictoken.ex_vectorize_return_call_two.end
+
+            self.assertIsInstance(result, np.ndarray)
+            correct = np.array([0., 0.4, 0.8, 1.2, 1.6, 2. ])
+            np.testing.assert_array_almost_equal(result, correct, -16)
```

Comment:
In most cases we use `assert_allclose()` - it's probably a little more tolerant than the check here - I fear this might be a little too stringent and introduce a spurious failure at some point in the future:

```suggestion
            np.testing.assert_allclose(result, correct)
```